src/nft/pages/collection/index.tsx

```diff
@@ -101,6 +101,10 @@ const Collection = () => {
         : isBagExpanded
         ? BAG_WIDTH
         : 0,
+    config: {
+      duration: 250,
```

Comment:
I think I saw in a theme figma doc somewhere a 125 or 150ms prescription for durations.

Reply:
I think we should stick with 250 here since it's such a large UI element changing - checked with Bryan and he signed off.